test/unit/lib/plugins/aws/provider.test.js

```diff
@@ -335,6 +329,58 @@ describe('AwsProvider', () => {
 });
 
 describe('test/unit/lib/plugins/aws/provider.test.js', () => {
+  it('`AwsProvider.getProviderName()` should resolve provider name', () => {
```

Comment:
Is there a reason as to why this test was moved verbatim and not refactored to use `runServerless`?

Reply:
My understanding is that a serverless run was not required in that case as we only use a static method from the aws provider. I'm no sure what would change when adding a runserverless a part of it.

Reply:
I took a look at the proposed spec in issue and it suggested to use `runServerless` as it's accessing `serverless` instance. In this case it doesn't make a big difference I guess, but I think we should stick to using the `runServerless`-based approach to be consistent here.

Reply:
Ok :+1: looks good to me. I will move it in a describe to have a common serverless run as indicated in the comments then :slightly_smiling_face: